apps/e2e/popovers-storybook-e2e/src/e2e/dropdown.component.cy.ts

```diff
@@ -0,0 +1,63 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('popovers-storybook', () => {
+  E2eVariations.forEachTheme((theme) => {
+    describe(`in ${theme} theme`, () => {
+      ['default', 'primary', 'link', 'disabled'].forEach((buttonStyle) => {
+        it(`should render ${buttonStyle} style dropdown buttons`, () => {
+          cy.visit(
+            `/iframe.html?globals=theme:${theme}&id=dropdowncomponent-dropdown--dropdown-${buttonStyle}-button`
+          );
+          cy.get('app-dropdown')
+            .should('exist')
+            .should('be.visible')
+            .screenshot(
+              `dropdowncomponent-dropdown--dropdown-${buttonStyle}-button-${theme}`
+            )
+            .percySnapshot(
+              `dropdowncomponent-dropdown--dropdown-${buttonStyle}-button-${theme}`,
+              {
+                widths: E2eVariations.DISPLAY_WIDTHS,
+              }
+            );
+        });
+      });
+      ['left', 'center', 'right'].forEach((horizontalAlignment) => {
```

Comment:
Can we add a line break before this? It took me a second to grasp this because it is up against the `describe` block before.